src/commands/git-metadata/git.ts

```diff
@@ -29,12 +29,12 @@ export const gitRemote = async (git: simpleGit.SimpleGit): Promise<string> => {
   for (const remote of remotes) {
     // We're trying to pick the remote called with the default git name 'origin'.
     if (remote.name === 'origin') {
-      return stripCredentials(remote.refs.push)
+      return remote.refs.push
```

Comment:
Moved in `getCommitInfo` directly to also apply it when an optional repositoryURL is passed as arg